bigtable/google/cloud/bigtable/row_data.py

```diff
@@ -194,8 +194,66 @@ class PartialRowsData(object):
 
     def __init__(self, response_iterator):
         self._response_iterator = response_iterator
+        self.generator = YieldRowsData(response_iterator)
+
         # Fully-processed rows, keyed by `row_key`
         self._rows = {}
+
+    def __eq__(self, other):
+        if not isinstance(other, self.__class__):
+            return False
+        return other._response_iterator == self._response_iterator
+
+    def __ne__(self, other):
+        return not self.__eq__(other)
+
+    @property
+    def state(self):
+        """State machine state.
+
+        :rtype: str
+        :returns:  name of state corresponding to currrent row / chunk
+                   processing.
+        """
+        return self.generator.state
+
+    @property
+    def rows(self):
+        """Property returning all rows accumulated from the stream.
+
+        :rtype: dict
+        :returns: row_key -> :class:`PartialRowData`.
+        """
+        # NOTE: To avoid duplicating large objects, this is just the
+        #       mutable private data.
+        return self._rows
```

Comment:
There is no point in having this property since `self._rows` is mutable (i.e. just upgrade the atrribute from `_rows` to `rows` if you want to do this)